tools/clang/lib/Sema/SemaChecking.cpp

```diff
@@ -8710,6 +8711,8 @@ namespace {
   };
 }
 
+#if 1 // HLSL Change Starts
```

Comment:
I know you just moved this here from down below.. but I wonder why they didn't just do `#if 0` rather than `#if 1 #else`

Reply:
I wondered the same, but thought to leave it as is. Perhaps there was code in the if 1 block before. I guess there's no reason to leave it as-is.